src/client/opamClient.ml

```diff
@@ -38,7 +38,7 @@ let orphans ?changes ?(transitive=false) t =
   let allnames = OpamPackage.names_of_packages all in
   let invalidated = Lazy.force t.invalidated in
   let universe =
-    OpamSwitchState.universe t ~requested:OpamPackage.Name.Set.empty Reinstall
+    OpamSwitchState.universe t ~requested:[] Reinstall
```

Comment:
I'm not too~ knowledgable but it seems to me that `OpamPackage.Set` is a better choice to replace `OpamPackage.Name.Set` (in place of a list). Could you change it that way?

Reply:
How about this?

Reply:
that looks like what i had in mind. Thanks a lot!